packages/@sanity/vision/src/components/VisionGui.tsx

```diff
@@ -694,7 +694,7 @@ export class VisionGui extends React.PureComponent<VisionGuiProps, VisionGuiStat
                   {API_VERSIONS.map((version) => (
                     <option key={version}>{version}</option>
                   ))}
-                  <option key="other" value={t('settings.other-api-version-label')}>
+                  <option key="other" value="other">
```

Comment:
Thanks Jørn, this is great! Please can you also change the `value` prop of the `Select` component to use the constant value `'other'` on line `690`?

```tsx
<Select
  value={customApiVersion === false ? apiVersion : 'other'}
  onChange={this.handleChangeApiVersion}
>
```

Without this change, the API Version select shows the incorrect value when "Other" is chosen.